resdk/tests/functional/permissions/e2e_permissions.py

```diff
@@ -1,6 +1,7 @@
 # pylint: disable=missing-docstring
 from resdk.exceptions import ResolweServerError
-from resdk.tests.functional.base import ADMIN_USERNAME, USER_USERNAME, BaseResdkFunctionalTest
```

Comment:
Should we change the above import to a relative import too?

Reply:
Yes. Will do.